tools/autograd/gen_autograd_functions.py

```diff
@@ -583,6 +603,7 @@ def save_var(var: SavedAttribute, is_output: bool) -> None:
                 )
             )
             should_append_raw_getsetdef = True
+            visit_name = f"{name}_"
         elif type == ListCType(OptionalCType(BaseCType(tensorT))):
             saved_variables.append(f"std::vector<SavedVariable> {name}_;")
```

Comment:
nit - replace all the `{name}_` with `{visit_name}`?

Reply:
The other uses aren't actually related to visitor patterns.